src/metabase/api/search.clj

```diff
@@ -18,35 +18,20 @@
             [metabase.models.pulse :refer [Pulse]]
             [metabase.models.segment :refer [Segment]]
             [metabase.models.table :refer [Table]]
-            [metabase.search :as search]
+            [metabase.search.config :refer :all]
```

Comment:
don't like the `:refer :all` here. would much prefer `:as search.config`

Reply:
:+1: fixed